superset/sql_lab.py

```diff
@@ -206,7 +206,8 @@ def execute_sql_statement(
         if SQL_MAX_ROW and (not query.limit or query.limit > SQL_MAX_ROW):
             query.limit = SQL_MAX_ROW
         if query.limit:
-            sql = database.apply_limit_to_sql(sql, query.limit)
+            #We are fetching one more than the requested limit in order to test whether there are more rows than the limit. 
```

Comment:
You'll need to split this into multiple lines to make the linter happy. I recommend adding a comment that later we'll drop the extra row:

```suggestion
            # We are fetching one more than the requested limit in order to test whether there are more rows than the limit. Later, the extra row will be dropped before sending the results back to the user.
```

Reply:
I just learned that python doesn't support multi-line comments.